bin/generate-stories.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs');
+
+const dextroseExtension = '.dextrose.tmp.js';
+
+const overRideStories = (dirPath, files, stringToFind, stringToReplace) =>
+  new Promise((resolve, reject) => {
+    files.forEach((file) => {
+      fs.readFile(file, 'utf8', (err, data) => {
+        if (err) {
+          reject(err);
+        }
+        if (data.includes(stringToFind)) {
+          const updatedData = data.replace(stringToFind, stringToReplace);
+          const trimmedFile = file.replace('.js', '');
+          const newFilePath = trimmedFile.concat(dextroseExtension);
+          fs.writeFileSync(`${dirPath}/${newFilePath}`, updatedData);
+        }
+      });
+    });
+    resolve();
+  });
+
+const findStoryFiles = (dir, matchers) => new Promise((resolve, reject) => {
+  fs.readdir(dir, 'utf8', (err, files) => {
+    if (err) reject(err);
+
+    if (!files) {
+      reject(Error('No files found'));
+    } else {
+      const storyFiles = files.map((file) => {
+        const isStoryFile = matchers.some(matcher => file.includes(matcher));
+        if (isStoryFile) {
+          return file;
+        }
+
+        return undefined;
```

Comment:
I don't think you need to return undefined, it should return undefined automatically.

Reply:
this is something that lint made me do :(